x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -151,6 +165,7 @@ class PackageClientImpl implements PackageClient {
     pkgName: string;
     pkgVersion?: string;
     spaceId?: string;
+    force?: boolean;
```

Comment:
is this needed?

Reply:
it's needed for reinstall, since that' what UI uses as well.

Reply:
I mean the force flag in `ensureInstalledPackage`, is it used?